README.rst

```diff
@@ -17,7 +17,7 @@ Testing locally
 ---------------
 
 To test locally in a Python virtual env, you will first need to install `GraphViz <http://graphviz.org/>`_
-On a Mac, this can be done via ``brew install graphviz``; on Ubuntu, use ``sudo apt install graphviz``.
+On a Mac, this can be done via ``brew install graphviz``; on Ubuntu, use ``sudo apt install graphviz``; on Redhat variants use ``sudo dnf install graphviz``.
```

Comment:
```suggestion
On a Mac, this can be done via ``brew install graphviz``; on Ubuntu, use ``sudo apt install graphviz``; on Red Hat variants use ``sudo dnf install graphviz``.
```